config/delta-producer/worship-posts/export.json

```diff
@@ -3,7 +3,8 @@
     {
```

Comment:
This file also has a quite different version on dev right now, some changes will probably be required after merge

Reply:
I've rebased it and applied the changes, could you check?